src/mumble/ServerInformation.ui

```diff
@@ -185,6 +188,25 @@
             </property>
            </widget>
           </item>
+          <item row="4" column="1">
+           <widget class="QLabel" name="qlVersionNote">
+            <property name="visible">
+              <bool>false</bool>
+            </property>
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="text">
+             <string>Warning: The server seems to report a truncated protocol version.</string>
```

Comment:
Maybe this should reference https://github.com/mumble-voip/mumble/issues/5827 in case a user wants to get more info on what's behind all this?

Reply:
Just the Issue number or the entire URL?

Reply:
It btw just occurred to me that the server will never report a truncated protocol version for itself. Only for clients. If the server itself is affected, it would report 1.5.18 or whatever the broken version was...

Reply:
> Just the Issue number or the entire URL?

Entire URL - which is what I posted but apparently GitHub was trying to be smart -.-


> It btw just occurred to me that the server will never report a truncated protocol version for itself. Only for clients. If the server itself is affected, it would report 1.5.18 or whatever the broken version was...

Indeed :thinking:

Reply:
I have removed the serverInformation changes and added a GitHub link to the UserInformation, but it reads "Issue #5827" instead of the full URL